### ISSUE TITLE
httpuv 1.6.10 crashes with "uv__finish_close: Assertion `handle->flags & UV_HANDLE_CLOSING' failed"

### ISSUE THREAD

**Issue by wfulp** [rstudio/httpuv#372]
I ran into a deployment issue in Posit Connect for an app where I upgraded the httpuv version to 1.6.10. The app looked fine locally but wouldn't show a network graph when deployed. I also got a cryptic error in the logs:

```
R: src/unix/core.c:258: uv__finish_close: Assertion `handle->flags & UV_HANDLE_CLOSING' failed.
```

This link has the exact same issue as my app (posted yesterday): https://community.rstudio.com/t/ui-completely-different-under-local-and-shinyapps-io-when-using-highcharter/165998/3

Both the linked example and my app deployed fine after I reverted to httpuv 1.6.9.

**Comment by jcheng5** [comment]
I'm so sorry--1.6.10 is bugged. It doesn't reproduce on all machines, but when it does it's fairly consistent. We're in the process of submitting a 1.6.11 release that will fix it.

Very sorry for the inconvenience. I hope you didn't spend too much time debugging.

**Comment by wch** [comment]
httpuv 1.6.11 is on CRAN now, so please try installing the new version.

**Comment by wfulp** [comment]
Both of my test apps deploy fine now with the new version. Thanks for such a quick update.

**Comment by jcheng5** [comment]
Note for folks on Windows or Mac, you may need to use `install.packages("httpuv", type="source")` to get the fixed 1.6.11 version, as the binary builds always run a few days behind the source builds.

**Comment by mjdzr** [comment]
I'm getting the same message from the same situation and I'm using httpuv version 1.6.14.
